proxy/proxy/apacheHandler.py

```diff
@@ -374,7 +374,7 @@ def handler(self, req):
         # All good; we expect ret to be an HTTP return code
         if not isinstance(ret, type(1)):
             raise rhnException("Invalid status code type %s" % type(ret))
-        log_debug(1, "Leaving with status code %s" % ret)
+        log_debug(2, "Leaving with status code %s" % ret)
```

Comment:
Not sure about this. If ret is "ok", then we should hide it. But a "404" or other errors should appear in the logs.
Are there other error messages displayed when this is "not ok" ?

Reply:
Yes, even if we set "2" as debug level here for this message, in case of "not ok", then we have the extra logging from `broker/rhnBroker.handler` still in place, so:

```
# before
>    3861 broker/rhnBroker.handler('Leaving handler with status code 404',)
>    3861 proxy/apacheHandler.handler('Leaving with status code 404',)
>   33063 proxy/apacheHandler.handler('Leaving with status code 0',)

# after
>    3861 broker/rhnBroker.handler('Leaving handler with status code 404',)
```

I think we should be good.

Reply:
Looks good